src/popup/constants/index.ts

```diff
@@ -1,5 +1,7 @@
 export const POPUP_WIDTH = 456;
 
+export const HEADER_HEIGHT = 120;
```

Comment:
👍

Reply:
I hope you're not commenting while on vacation 👀

Reply:
lolol naw i'm back today